.github/workflows/push.yml

```diff
@@ -658,19 +658,13 @@ jobs:
           aws-secret-access-key: ${{ secrets.ECR_AWS_SECRET_ACCESS_KEY }}
           aws-region: eu-west-1
 
-      - name: Docker build image
+      - name: Comment what is being deployed in PR
         working-directory: infra
         run: |
-          echo Registry is: ${{env.DOCKER_BASE_IMAGE_REGISTRY}}
-          echo Image tag is: ${{env.NODE_IMAGE_TAG}}
-          export EXTRA_DOCKER_BUILD_ARGS="--build-arg DOCKER_IMAGE_REGISTRY=${{env.DOCKER_BASE_IMAGE_REGISTRY}} --build-arg NODE_IMAGE_TAG=${{env.NODE_IMAGE_TAG}}"
-          ./scripts/build-docker-container.sh "$DOCKER_TAG"
+          yarn install --immutable
```

Comment:
Does this add overhead or are we doing the same thing in the docker build?

Reply:
As I read the script and Dockerfile, this is doing the same, but with fewer steps, and less complexity.

Reply:
Nvm, the image is used in the feature-deploy Spinnaker pipeline
https://github.com/island-is/helm/blob/4c8de03253282ec394c480d8e0f2da079acf7fb1/pipelines/stages/feature-generate.json#L48